### PR TITLE
💄 Style Global CSS

### LINKED ISSUE #2: Style Global CSS
- [x] Set global Colors
- [x] Set Fonts an Fontsizes

### PR DESCRIPTION
**Pull request by sweisz**
Fixes #2
- Add Colors
- Add Font Links
- Set FontSizes

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+}
+
+@font-face {
+  font-family: "Copperplate";
+  src: url("../public/fonts/Copperplate/Copperplate.woff") format("woff");
+}
+
+:root {
+  --accent-color: #ff4064;
+  --second-accent-color: #ffb700;
```
Comment (fre-ben) [inline styles/globals.css]: Small suggestion: Maybe name these `primary` and `secondary`, so it matches with the other variables

**Review by fre-ben — APPROVED** [review]
LGTM 🤖 
Just a small suggestion.

**Review by DNaunin — APPROVED** [review]
GJ! 🎉

**Review by christophvogel — APPROVED** [review]
LGTM 👍

File: styles/globals.css (hunk 1 above)
Comment (sweisz) [reply styles/globals.css]: Yes tx ;)

File: styles/globals.css [hunk 2]
```diff
@@ -1,9 +1,33 @@
+@font-face {
+  font-family: "DIN";
+  src: url("../public/fonts/DIN/DIN-Regular.woff") format("woff");
```
Comment (lmachens) [inline styles/globals.css]: `public` is the root folder, thus, you can access files like this.
See https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook
```suggestion
  src: url("/fonts/DIN/DIN-Regular.woff") format("woff");
```

File: styles/globals.css [hunk 3]
```diff
@@ -14,3 +38,16 @@ a {
...
+}
+.StartPageHeadText {
+  font-family: "SignPainter-HouseScript", sans-serif;
+  font-size: 4em;
+}
+.StartPageSubHeadline,
+.SkillPageSubHeadline {
+  font-family: "Copperplate", serif;
+  font-size: 1em;
+}
```
Comment (lmachens) [inline styles/globals.css]: I don't think these classes should be part of your `globals.css`

**Review by lmachens — COMMENTED** [review]
gj

**Review by lmachens — APPROVED** [review]
gj 🚀

**Pull request merged**